### ISSUE TITLE
render: execute template: template: no template "gh-ownership.yaml" associated with template "_gotpl_default"

### ISSUE THREAD

**Issue by xino12** [bluebrown/go-template-cli#19]
I'm seeing this error when executing with the latest release, but it works correctly with 0.2.0.

tpl -f partials/ops_jira_ticket_id.yaml -f templates/gh-ownership.yaml --name gh-ownership.yaml > output/gh-ownership.yaml 
render: execute template: template: no template "gh-ownership.yaml" associated with template "_gotpl_default"

What configuration I'm missing?

**Comment by bluebrown** [comment]
Ok I get the same with the repo test data

```bash
& tpl -f cmd/tpl/testdata/apple.tpl -f cmd/tpl/testdata/mango.tpl -n mango.tpl
render: execute template: template: no template "mango.tpl" associated with template "_gotpl_default"
```

Also this should not work as the --name flag would be required.

```bash
tpl -f cmd/tpl/testdata/apple.tpl -f cmd/tpl/testdata/mango.tpl
<no value>
```

**Comment by bluebrown** [comment]
I have added 2 failing tests. Now lets debug.

**Comment by bluebrown** [comment]
The bug is here https://github.com/bluebrown/go-template-cli/blob/90a856d378e44b42f3278350da0d28696bd9a923/cmd/tpl/cli.go#L137

```go
#&{file f template file path. Can be specified multiple times [cmd/tpl/testdata/apple.tpl,cmd/tpl/testdata/mango.tpl] [] true   false  map[]}
```

f is only visited a single time, and its value is just the slice, logically. So that change that was supposed to fix a bug, made the bug worse.

**Comment by bluebrown** [comment]
@xino12 , this should be fixed in 0.3.1. Thanks for flagging. Please report back if you still have problems.
